packages/support/resources/views/components/input/wrapper.blade.php

```diff
@@ -174,6 +175,27 @@
                 />
             @endif
 
+            @if ($revealable)
+                <div class="{{ $actionsClasses }}">
+                    <x-filament::icon-button
+                        color="gray"
+                        icon="heroicon-o-eye"
+                        icon-alias="forms::components.text-input.show-password"
+                        x-on:click="inputType = 'text'"
+                        x-show="inputType === 'password'"
+                    />
+
+                    <x-filament::icon-button
+                        color="gray"
+                        icon="heroicon-o-eye-slash"
+                        icon-alias="forms::components.text-input.hide-password"
+                        x-cloak
+                        x-on:click="inputType = 'password'"
+                        x-show="inputType !== 'password'"
+                    />
+                </div>
+            @endif
+
             @if (count($suffixActions))
                 <div class="{{ $actionsClasses }}">
                     @foreach ($suffixActions as $suffixAction)
```

Comment:
Shouldn't the actions actually be moved to here, @Stichoza? 🤔

Reply:
Correct, changed.